.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Unit tests, QA
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Here too the '**' to target all branches' PR. Or could be considered 🤔.

 Also maybe you can use common YAML syntax for lists. Or do you prefer the abbreviated way? https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

Reply:
What do you think of the YAML syntax?
branches:
  - main